src/mscorlib/shared/System/Collections/Generic/Dictionary.cs

```diff
@@ -759,6 +759,24 @@ IEnumerator IEnumerable.GetEnumerator()
             return new Enumerator(this, Enumerator.KeyValuePair);
         }
 
+        /// <summary>
+        /// Ensures that the capacity of this dictionary is at least the given minimum value
```

Comment:
I would say `Ensures that the dictionary can hold up to 'capacity' entries without any further expansion of its backing storage.`